### ISSUE TITLE
Can't use the Support Library yet!

### ISSUE THREAD

**Issue by lewismcgeary** [lewismcgeary/AndroidtoAppleVectorLogo#8]
[As stated in Chris Banes article](https://medium.com/@chrisbanes/appcompat-v23-2-age-of-the-vectors-91cbafa87c88#.t3rgfm6zd) as of Support Library 23.2.0 Path Morphing is not available in the Support Library version of AnimatedVectorDrawable

**Comment by AndroidDeveloperLB** [comment]
Correct. Adding the "vectorDrawables.useSupportLibrary = true" line doesn't help for this.

What is exactly supported on pre-API 21 exactly ? They only mention what's not supported...
Also, are there any workarounds or alternative libraries for this?

**Comment by lewismcgeary** [comment]
I assume everything else is supported, but who knows... ¯_(ツ)_/¯
This other example of mine is compatible pre-API 21: https://github.com/lewismcgeary/AnimatedVectorDrawableCompat-play-to-reset-button

As for other libraries, there is [vector-compat](https://github.com/wnafee/vector-compat). It looks like it's no longer being actively developed but seems to support path morphing (I haven't used it personally).

**Comment by AndroidDeveloperLB** [comment]
Nice.
Can you please make a tiny sample that shows all possible features we can use for Vector animations? And also show which of them are supported by the support library and which aren't?
I don't even care if it looks cool like here.

**Comment by lewismcgeary** [comment]
It's an interesting idea to have a sample demonstrating them all, but it wouldn't be tiny, there's quite a few. You can see from the [first part of the VectorDrawable documentation](https://developer.android.com/reference/android/graphics/drawable/VectorDrawable.html) all the properties they have, most of which can be animated. 

Not something I have time to do right now, but will hopefully put out more in the future!

**Comment by AndroidDeveloperLB** [comment]
Please let me know if you do it. It can help understanding how to use it.
